packages/dds/tree/README.md

```diff
@@ -11,15 +11,10 @@ and some of the implications of those goals for developers working on this packa
 
 Notable consideration that early adopters should be wary of:
 
--   The persisted format is not stable and long term support for persisted format is not yet in effect.
-    This means clients running more recent versions of SharedTree may find themselves unable to load old document or may corrupt them when editing.
-    Current estimate for LTS is April 2024.
--   SharedTree currently has unbounded memory growth:
-    -   Removed content is retained forever.
```

Comment:
Until [https://github.com/microsoft/FluidFramework/pull/21372/](https://github.com/microsoft/FluidFramework/pull/21372/) is merged, removed content will keep on being retained forever.

Reply:
I have updated the text to mention this.